src/styles/_about.scss

```diff
@@ -77,7 +77,7 @@
     #process {
       background: $finsec-royal-blue;
       > div {
-        padding: 150px 20px;
+        padding: 150px 20px 100px;
```

Comment:
Yes, it seems to work fairly well - spacing the <p class="disclaimer"> to a healthy distance inside the section